src/main/java/uk/gov/cabinetoffice/di/ipv/dcsatpservice/DcsAtpServiceApplication.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.cabinetoffice.di.ipv.dcsatpservice;
```

Comment:
I think we need to have cabinetoffice in the package, a root of `uk.gov.di.ipv` should be fine for our stuff.

Reply:
Ok, will change it out to a shorter package name